src/protocols/interaction_model/Constants.h

```diff
@@ -62,6 +62,49 @@ enum class MsgType : uint8_t
     TimedRequest          = 0x0a,
 };
 
+//TODO: Cleanup Reserved protocolCode from ember
+enum class ProtocolCode : uint16_t
```

Comment:
Please document that this table comes from the IM's "Status Code Table" section from the spec.

Reply:
Done, thanks